base/docs/utils.jl

```diff
@@ -38,6 +38,12 @@ show(io::IO, ::MIME"text/html", h::HTML{<:Function}) = h.content(io)
     @html_str -> Docs.HTML
 
 Create an `HTML` object from a literal string.
+
+# Examples
+```jldoctest
+julia> @html_str "Julia"
```

Comment:
```suggestion
julia> html"Julia"
```

Reply:
Something like this might explain the purpose of `@html_str` a bit better:
```suggestion
julia> show(stdout, MIME("text/html"), html"Julia")
```